aspnetcore/fundamentals/ihostedservice.md

```diff
@@ -29,50 +29,59 @@ Hosted services implement the [IHostedService](/dotnet/api/microsoft.extensions.
 
 * [StartAsync(CancellationToken)](/dotnet/api/microsoft.extensions.hosting.ihostedservice.startasync) - Called after the server has started and [IApplicationLifetime.ApplicationStarted](/dotnet/api/microsoft.aspnetcore.hosting.iapplicationlifetime.applicationstarted) is triggered. `StartAsync` contains the logic to start the background task.
 
-* [StopAsync(CancellationToken)](/dotnet/api/microsoft.extensions.hosting.ihostedservice.stopasync) - Triggered when the host is performing a graceful shutdown. `StopAsync` contains the logic to end the background task and dispose of any unmanaged resources.
+* [StopAsync(CancellationToken)](/dotnet/api/microsoft.extensions.hosting.ihostedservice.stopasync) - Triggered when the host is performing a graceful shutdown. `StopAsync` contains the logic to end the background task and dispose of any unmanaged resources. If an error is thrown during background task execution, `StopAsync` might not be called.
```

Comment:
I meant if the application shuts down un-gracefully due to an error StopAsync may not be called.

Reply:
I felt like we could give them one (obvious) example. If u have a better example, I can swap that in there.

> If the app shuts down ungracefully (for example, the app's process fails), \`StopAsync\` might not be called.